bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewWebTargets.java

```diff
@@ -94,8 +94,13 @@ private Response invoke(Invocation invocation, WebTarget target) {
             response = invocation.invoke();
         }
 
-        if (response.getStatus() != 200) {
-            logger.error("Bridge returned {} while invoking {} : {}", response.getStatus(), target.getUri(),
+        if (response.getStatus() == 423) {
+            // the hub seems to return a 423 error (resource locked) once per day around midnight
+            // this is probably some kind of regular re-initialization process, so suppress the error log
+            logger.debug("Bridge returned '423' while invoking {}", target.getUri());
+            return null;
+        } else if (response.getStatus() != 200) {
+            logger.error("Bridge returned '{}' while invoking {} : {}", response.getStatus(), target.getUri(),
```

Comment:
```suggestion
            logger.warn("Bridge returned '{}' while invoking {} : {}", response.getStatus(), target.getUri(),
```

Or maybe even less like `INFO`.  Depends if this can "just happen" and thebinding recovers from that or if the user has to interact.

Reply:
Ok. Done.